deno_dist/helper/ssg/index.ts

```diff
@@ -1,9 +1,40 @@
 import { Buffer } from "node:buffer";
-import * as path from 'node:path'
 import { inspectRoutes } from '../../helper/dev/index.ts'
 import type { Hono } from '../../hono.ts'
 import type { Env, Schema } from '../../types.ts'
 
+export const dirname = (path: string) => {
```

Comment:
How about moving the `dirname` and `joinPaths` to `utilts.ts`?

Reply:
I think it's nice idea.
In this case, Should I put files outside `src/helper/ssg`? If I put this code outside `src/helper/ssg`, other codes can easily imports `dirname` and `joinPaths` functions. If you don't need this feature, I put this functions to `src/helper/ssg/utils.ts`.

Reply:
I think `src/helper/ssg/utils.ts` is better.

If you don't plan to use it in other places right now, it is better to make it exclusive to this feature. You can make changes without worrying about the impact on others.

Reply:
I understand. I moved that functions.